CMakeLists.txt

```diff
@@ -367,27 +367,37 @@ if(WITH_CUDA)
   include("${PCL_SOURCE_DIR}/cmake/pcl_find_cuda.cmake")
 endif()
 
+
+# Find VTK
```

Comment:
Please mention that the location is specific (VTK before Qt) and the reasons for the same